### PR TITLE
COMMERCE-9282, COMMERCE-9283, COMMERCE-9284, COMMERCE-9264, COMMERCE-9265 | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/COMMERCE-9282
https://issues.liferay.com/browse/COMMERCE-9283
https://issues.liferay.com/browse/COMMERCE-9284
https://issues.liferay.com/browse/COMMERCE-9264
https://issues.liferay.com/browse/COMMERCE-9265

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

**Comment by Kvale1733** [comment]
It looks like the image gallery test failed because the order of the images were different on CI. Or maybe I needed to update not sure. Either way I'm updating the test and running the suite again.

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

**Comment by Kvale1733** [comment]
It seems the order of the images shown on the gallery are random.
CommerceDisplayPageTemplates#CanViewProductImagesUsingImageGalleryFragment depends on consistency in these images. I will have to figure something else out.

**Comment by Kvale1733** [comment]
This is why the test keeps failing.

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

**Comment by Kvale1733** [comment]
All the tests are passing now

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDisplayPageTemplates.testcase [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+
+			AssertElementPresent(
+				key_fsProductImage = "",
+				locator1 = "CommerceFrontStore#FS_IMAGE_GALLERY_MAIN_IMAGE");
+
+			Click(locator1 = "CommerceFrontStore#FS_IMAGE_GALLERY_ARROW_PREV");
+
+			AssertElementPresent(
+				key_fsProductImage = "",
+				locator1 = "CommerceFrontStore#FS_IMAGE_GALLERY_MAIN_IMAGE");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDisplayPageTemplates.testcase:455]: @Kvale1733 why are these 2 `key_fsProductImage` fields empty?

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Pull request closed without merging**